### PR TITLE
linter: Enable linter on lib 

### PR DESCRIPTION
**Pull request by exoego**
# Purpose

To improve the code quality of actual library sources.

# Overview

- Enabled standardb on `lib` sources but suppressed most of conventional-level checks. Will improve eventually.
- Fixed one warning-level chekc [Lint/Loop](https://docs.rubocop.org/rubocop/cops_lint.html#lintloop)

# Checklist

- [x] Code compiles correctly
- [x] Created tests which fail without the change (if possible)
- [x] All tests passing
- [x] Extended the README / documentation, if necessary

### CODE REVIEW

File: .standard_todo.yml [hunk 1]
```diff
@@ -1,6 +1,32 @@
 ---
 ignore:
-  - 'lib/trino/client/*.rb'
```
Comment (yuokada) [inline .standard_todo.yml:3]: I'm not sure why the target files are expanded.

I think it's enough to add the rules to the current target.

File: .standard_todo.yml [hunk 2]
```diff
@@ -1,6 +1,32 @@
...
+  - 'lib/**/*.rb':
```
Comment (exoego) [inline .standard_todo.yml:3]:
```suggestion
  - 'lib/trino/client/*.rb':
```

File: .standard_todo.yml [hunk 3]
```diff
@@ -1,6 +1,32 @@
 ---
 ignore:
-  - 'lib/trino/client/*.rb'
+  - 'lib/trino/client/*.rb':
```
Comment (exoego) [inline .standard_todo.yml:3]:
```suggestion
 - 'lib/trino/client/*.rb':
```

File: .standard_todo.yml (hunk 1 above)
Comment (exoego) [reply .standard_todo.yml:3]: revereted

File: .standard_todo.yml [hunk 4]
```diff
@@ -1,6 +1,32 @@
 ---
 ignore:
-  - 'lib/trino/client/*.rb'
+ - 'lib/trino/client/*.rb':
```
Comment (exoego) [inline .standard_todo.yml:3]:
```suggestion
  - 'lib/trino/client/*.rb':
```

**Review by yuokada — APPROVED** [review]
Looks good to me ✅

**Pull request merged**